clients/drcachesim/drpt2trace/pt2ir.cpp

```diff
@@ -300,11 +315,23 @@ pt2ir_t::convert(OUT instrlist_t **ilist)
             decode(GLOBAL_DCONTEXT, insn.raw, instr);
             instr_set_translation(instr, (app_pc)insn.ip);
             instr_allocate_raw_bits(GLOBAL_DCONTEXT, instr, insn.size);
+            instr_set_isa_mode(instr,
+                               insn.mode == ptem_32bit ? DR_ISA_IA32 : DR_ISA_AMD64);
+#ifdef DEBUG
+            /* XXX: Currently, the PT raw data may contain 'STAC' and 'CLAC'
```

Comment:
reference the actual issue number in a TODO i#NNN

Reply:
Done.